controllers/grants/views/recipient-detail.njk

```diff
@@ -4,122 +4,126 @@
+                            <header class="card__header">
+                                <h3 class="card__title">{{ copy.recipientDetail.awardingProgrammes }}</h3>
+                                <div class="card__body">
+                                    {% for programme in recipientProgrammes %}
+                                        <a href="{{ pgLink('programme', programme.label) }}"
+                                           class="related-programme">
+                                            {{ programme.label }}
+                                        </a>
+                                    {% endfor %}
+                                </div>
+                            </header>
+                        </aside>
+                    </div>
 
+                </div>
             </div>
-        </div>
 
-        <div class="u-inner-wide-only">
-            <h2>
-                {{ __('funding.pastGrants.search.recipientDetail.numFundedTotal', totalResults, totalAwarded) }}
-            </h2>
-            <div class="flex-grid">
-                {% for grant in recipientGrants %}
-                    <div class="flex-grid__item">
-                        <div class="card u-no-margin">
-                            <header class="card__header">
-                                <h3 class="card__title u-no-margin">
-                                    <a href="{{ localify("/funding/grants/" + grant.id) }}">
-                                        {{ grant.title }}
-                                    </a>
-                                </h3>
-                            </header>
-                            <div class="card__body">
-                                <p class="project-caption">£{{ grant.amountAwarded | numberWithCommas}}
-                                    on {{ formatDate(grant.awardDate, DATE_FORMATS.short) }}</p>
-                                <dl class="o-definition-list o-definition-list--compact u-margin-top-s project-details">
-                                    <dt>{{ copy.grantDetail.fields.programme }}</dt>
-                                    {% for programme in grant.grantProgramme %}
-                                        <dd>
-                                            <a href="{{ pgLink('programme', programme.title) }}">{{ programme.title }}</a>
-                                            {% if grant.isActive %}
-                                                <span class="u-pull-right active-badge">{{ copy.grantDetail.statuses.activeProject }}</span>
-                                            {% endif %}
-                                        </dd>
-                                    {% endfor %}
-                                </dl>
-                            </div>
+            <div class="u-inner-wide-only">
+                <h2>
+                    {{ __('funding.pastGrants.search.recipientDetail.numFundedTotal', totalResults, totalAwarded) }}
```

Comment:
Whilst we're tweaking this, any chance you could make this use `__n` to pluralise it, i.e. 10 projects, rather than 10 project(s)